wdl/tasks/Longbow.wdl

```diff
@@ -176,8 +156,8 @@ task Peek {
         disk_gb:            disk_size,
         boot_disk_gb:       10,
         preemptible_tries:  1,
-        max_retries:        0,
-        docker:             "us.gcr.io/broad-dsp-lrma/lr-longbow:0.5.33"
+        max_retries:        1,
+        docker:             "us.gcr.io/broad-dsp-lrma/lr-longbow:0.5.34"
```

Comment:
Can you check in this repo that 0.5.33 is no longer used anywhere?
Avoid fragmentation.

Reply:
Yup - `0.5.33` is no longer used anywhere.